### PR TITLE
Fe gatsby setup

### PR DESCRIPTION
**Pull request by beasterbro**
Frontend dependencies and directories setup

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,74 @@ node_modules/
...
+
+# dotenv environment variable files
+.env*
+
+# gatsby files
+.cache/
+public
+
+# Mac files
+.DS_Store
```
Comment (lucaspopp0) [inline .gitignore:76]: Already in there

File: services/frontend/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
+  },
+  "dependencies": {
+    "gatsby": "^2.24.54",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0"
+  },
+  "devDependencies": {
+    "gatsby": "^2.24.60",
+    "prettier": "2.1.1"
```
Comment (lucaspopp0) [inline services/frontend/package.json:23]: Not needed here since it's installed globally

File: services/frontend/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "gatsby-starter-hello-world",
```
Comment (lucaspopp0) [inline services/frontend/package.json:2]: Thoughts on calling it `@logan/web-frontend` for consistency?

File: services/frontend/package.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  "private": true,
+  "description": "A simplified bare-bones starter for Gatsby",
+  "version": "0.1.0",
+  "license": "0BSD",
+  "scripts": {
+    "build": "gatsby build",
```
Comment (lucaspopp0) [inline services/frontend/package.json:8]: Can we add a README.md to explain what these do?

File: services/frontend/package.json (hunk 2 above)
Comment (lucaspopp0) [reply services/frontend/package.json:23]: I do think there are specific ESLint configurations for React though that we might wanna set up. I can help configure that if you want

**Review by lucaspopp0 — CHANGES_REQUESTED** [review]

File: services/frontend/package.json (hunk 3 above)
Comment (beasterbro) [reply services/frontend/package.json:2]: Fixing it now, that was an error for the script used to generate the file.

File: services/frontend/package.json (hunk 4 above)
Comment (beasterbro) [reply services/frontend/package.json:8]: I'll type one up now, but most of them or for internal use in gatsby.

File: services/frontend/package.json (hunk 2 above)
Comment (beasterbro) [reply services/frontend/package.json:23]: Sure, I haven't been getting issues, but a bit more detailed style guide wouldn't hurt

**Review by lucaspopp0 — APPROVED** [review]

**Pull request merged**